### PR TITLE
feat: more styling of enter email step

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/utils/components/Button.tsx [hunk 1]
```diff
@@ -1,9 +1,18 @@
 import type { FC } from "hono/jsx";
+import cn from "classnames";
 
-const Button: FC<{ children: string | string[] }> = ({ children }) => {
+type Props = {
+  children: JSX.Element[] | JSX.Element | string | string[];
```
Comment (danielgent) [inline src/utils/components/Button.tsx:5]: This is a bit annoying :smile: 

There might be a better type that encompases all these

File: src/utils/components/Button.tsx [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+  className?: string;
+};
+
+const Button: FC<Props> = ({ children, className }) => {
   return (
     <button
-      class="relative w-full rounded-lg text-center px-4 py-5 bg-primary text-textOnPrimary hover:bg-primaryHover"
+      class={cn(
+        "relative w-full rounded-lg text-center px-4 py-5 bg-primary text-textOnPrimary hover:bg-primaryHover",
+        className,
```
Comment (danielgent) [inline src/utils/components/Button.tsx:14]: I probably mentioned this elsewhere!  as this isn't React we use `class` in the JSX *BUT* we can't have a variable called class in JS, so I'm falling back to the React standard

_I wonder_ if tailwind picks this up?  We've had issues before with what tailwind does and doesn't pick up (in order to add new classes as per the changed CSS above)

I'll find out soon enough though as I'm iterating to match login2

File: src/utils/components/Button.tsx (hunk 1 above)
Comment (markusahlstrand) [reply src/utils/components/Button.tsx:5]: I guess we could make our own type? But agree...

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**